src/screens/Asset/ReceiveModal.js

```diff
@@ -117,24 +118,35 @@ const ReceiveModal = ({
       <ContentWrapper forceInset={{ top: 'never', bottom: 'always' }}>
         <InfoView>
           {!!ensName && (
-            <TextWithCopy textToCopy={ensName} toastText={t('toast.ensNameCopiedToClipboard')} iconColor={colors.link}>
+            <TextWithCopy
+              textToCopy={ensName}
+              toastText={t('toast.ensNameCopiedToClipboard')}
+              iconColor={colors.link}
+              textStyle={styles.ensName}
+            >
```

Comment:
You still need to apply `adjustFontSizeToFit` here (and expose it on `TextWithCopy` component), as it breaks without ens name,

Reply:
Done.